src/modules/dashboard/components/ColonyHome/ColonyUnclaimedTransfers/ColonyUnclaimedTransfers.tsx

```diff
@@ -0,0 +1,169 @@
+
+  const { data: tokenData } = useTokenQuery({
+    variables: { address: firstItem?.token || '' },
+  });
+
+  const transform = useCallback(
+    mergePayload({ colonyAddress, tokenAddress: firstItem?.token || '' }),
+    [colonyAddress, firstItem],
+  );
+
+  if (error) console.warn(error);
+
+  if (loading) {
+    return (
+      <MiniSpinnerLoader
+        className={styles.main}
+        loadingText={MSG.loadingData}
+      />
+    );
+  }
+
+  const token = tokenData?.token;
+
+  return data && data.processedColony.unclaimedTransfers.length ? (
+    <div className={styles.main}>
+      <Heading appearance={{ size: 'normal', weight: 'bold' }}>
+        <NavLink to={`/colony/${colonyName}/funds`}>
+          <FormattedMessage {...MSG.title} />
+        </NavLink>
+      </Heading>
+      {data && (
+        <ul>
+          <li className={styles.firstLineContainer}>
+            <div className={styles.tokenItem}>
+              <span className={styles.tokenValue}>
+                <Numeral
+                  unit={getTokenDecimalsWithFallback(token?.decimals)}
+                  value={firstItem?.amount || ''}
+                />
+              </span>
+              <span className={styles.tokenSymbol}>
+                {token?.symbol ? (
+                  <span>{token?.symbol}</span>
+                ) : (
+                  <FormattedMessage {...MSG.unknownToken} />
+                )}
+              </span>
+            </div>
+            <Tooltip
+              appearance={{ theme: 'dark', size: 'medium' }}
+              trigger="hover"
+              content={
+                <div className={styles.tooltip}>
+                  <FormattedMessage {...MSG.tooltip} />
+                </div>
+              }
+              /*
+                Not showing arrow here.
+                If the screen is narrow the tooltip gets moved and the arrow looks wacky.
+              */
+              showArrow={false}
+              placement="top-start"
+              popperProps={{
+                modifiers: [
+                  {
+                    name: 'offset',
+                    options: {
+                      offset: [5, 8],
+                    },
+                  },
+                ],
+              }}
+            >
+              <ActionButton
+                text={MSG.claimButton}
+                className={styles.button}
+                submit={ActionTypes.COLONY_CLAIM_TOKEN}
+                error={ActionTypes.COLONY_CLAIM_TOKEN_ERROR}
+                success={ActionTypes.COLONY_CLAIM_TOKEN_SUCCESS}
+                transform={transform}
+                disabled={!isNetworkAllowed || !hasRegisteredProfile}
+              />
+            </Tooltip>
+          </li>
+          <li>
+            <div className={styles.tokenItem}>
+              +{data?.processedColony.unclaimedTransfers.length - 1}
+              <FormattedMessage {...MSG.more} />
+            </div>
```

Comment:
Please refactor this to make use of the message descriptor's value prop.

So something like this:
```suggestion
            <div className={styles.tokenItem}>
              <FormattedMessage {...MSG.more} values={{ extraClaims: data?.processedColony?.unclaimedTransfers?.length - 1 }} />
            </div>
```
Maybe even, extract the extra claim value into a separate variable to make it even more readable:
```js
const extraClaims = data?.processedColony?.unclaimedTransfers?.length - 1;
```

And then, inside the message descriptor, you do something like this:

```jsx
  more: {
    id: 'dashboard.ColonyHome.ColonyUnclaimedTransfers.more',
    defaultMessage: `+{extraClaims} more`,
  },
```